controllers/agentmachine_controller.go

```diff
@@ -119,11 +119,6 @@ func (r *AgentMachineReconciler) Reconcile(ctx context.Context, req ctrl.Request
 		return *res, err
 	}
 
-	// If the AgentMachine is ready, we have nothing to do
-	if agentMachine.Status.Ready {
-		return ctrl.Result{}, nil
-	}
```

Comment:
I didn't see any other controller that changed this field, so I thought it could prevent reconciling the agentMachine since nothing changes it back to not ready.